Assets/Scripts/Robot/Sonar.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+
+public class Sonar : MonoBehaviour
+{
+    //Script counts all distances from objects in tasks
+
+    public float viewRadious;
+    public Transform tasks;
+    private Dictionary<Transform, int> children;
```

Comment:
Apply property, to easily get objects that are in area, Rename variable to more descriptive ex: objectsInSonarArea